extensions/rich_text_components/Svgeditor/directives/oppia-noninteractive-svgeditor.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the oppia noninteractive svg editor component.
+ */
+
+import { UpgradedServices } from 'services/UpgradedServices';
```

Comment:
Can you check if this import is really required?

Reply:
It is not required and I have removed it. Done